be/src/util/uid_util.h

```diff
@@ -97,22 +97,11 @@ struct UniqueId {
     }
 
     // std::map std::set needs this operator
-    bool operator<(const UniqueId& right) const {
-        if (hi != right.hi) {
-            return hi < right.hi;
-        } else {
-            return lo < right.lo;
-        }
-    }
+    auto operator<=>(const UniqueId& right) const = default;
```

Comment:
suggest keep this function, the default behavior depends on the memory layout of the member variables.

Reply:
But the memory layout of the class members is determined?

Reply:
if new field is added, or code refactored, it is easy to cause issues without even noticed. probably UT shall cover the comparison behavior.